Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -51,6 +51,30 @@
       - id: HoloprojectorSecurity
       - id: ClothingEyesHudSecurity
 
+- type: entity
```

```suggestion
#SS220-Pilot
- type: entity
```
Стоит добавлять комменты, чтобы в будущем можно было легко понять, что делали именно мы.